tests/unit/cloud/clouds/test_proxmox.py

```diff
@@ -162,3 +162,39 @@ def test__authenticate_with_custom_port(self):
                 verify=True,
                 data={"username": ("fakeuser",), "password": "secretpassword"},
             )
+
+    def test__import_api_version_6(self):
+        """
+        Test successful import of version 6 api
+        """
+        with patch("salt.cloud.clouds.proxmox.requests.get") as mock_request:
+            url = "proxmox"
+            port = "8006"
+            verify_ssl = False
+
+            mock_request.return_value.text = """
+var pveapi = [
+    {
+        "testkey": "testvalue"
+    }]
+;
+"""
+            proxmox._import_api()
```

Comment:
**suggestion** this *could* be written like

```suggestion
            proxmox._import_api()
            assert proxmox.api == [{"testkey": "testvalue"}]
```

The same for the below test. I *did* run the tests locally without your code changes, and saw failures. I think the assertions would be somewhat worthwhile. If you want to make the changes, :+1: otherwise, I'll get this in the merge queue tomorrow.

Reply:
Thanks for taking a look. I won't have time to review/change anything here today but can clean it up a bit more this weekend if needed